lua/tardis-nvim/init.lua

```diff
@@ -103,19 +94,17 @@ local function setup_keymap(origin, buffers)
     end
 end
 
-local function setup_autocmds(origin, buffers)
+local function setup_autocommands(buffers, origin)
     local group = vim.api.nvim_create_augroup('Tardis', {})
     for _, buffer in ipairs(buffers) do
         vim.api.nvim_create_autocmd({'BufUnload'}, {
             group = group,
             callback = function()
                 vim.api.nvim_del_augroup_by_id(group)
-                local current_buffer = vim.api.nvim_get_current_buf()
+                local to_close = vim.tbl_filter(function(other) return other.fd ~= buffer.fd end, buffers)
```

Comment:
Considering that this is pretty similar to my updated solution (although cleaner, I will admit), I can just add this to my PR and then you can merge everything pretty easily. Unless there are still changes in my PR that you don't like.

Reply:
That'd be possible too, however, to keep the history clean tidy I'd like to drop these two commits:
a937f85c8077895ce8beb196d5ba5b732de47c8e
bfa07a7e8ffdc42b8118b1168d601abb311e9a39. 

If you can pull those two out, I could merge yours, and then merge this afterwards